### PR TITLE
[ProductBundle] Update installation instructions for v0.13

### PR DESCRIPTION
**Pull request by adamelso**
Q  | A
----|----
Doc fix? | yes
New docs? | no
Fixed tickets | https://github.com/Sylius/Sylius/issues/2480

There may be more updates done later to restore documentation regarding routing. Since the last update, the routing configuration file was replaced by the CMF Router.

### CODE REVIEW

**Comment by adamelso** [comment]
Will this be merged? My motivation for this PR was the result of this [StackOverflow question](http://stackoverflow.com/q/28731801/558866).

File: bundles/SyliusProductBundle/installation.rst [hunk 1]
```diff
@@ -60,28 +65,62 @@ Put this configuration inside your ``app/config/config.yml``.
...
 
-Routing configuration
----------------------
 
-Add the following to your ``app/config/routing.yml``.
+Then configure Doctrine extensions which are used by the bundle.
 
 .. code-block:: yaml
 
-    sylius_product:
```
Comment (aRn0D) [inline bundles/SyliusProductBundle/installation.rst]: why did you remove that ?

Comment (adamelso) [inline bundles/SyliusProductBundle/installation.rst]: Because there is no `@SyliusProductBundle/Resources/config/routing.yml` resource - this was removed as routing for the product is now managed by the CMF router. We'll need to document the changes made by @lsmith77 in https://github.com/Sylius/Sylius/commit/4488efd0cff364047f307711e2de513500d271ac (unfortunately the CMF router is still a bit of a black box to me).

**Comment by adamelso** [comment]
Closing this for now since Sylius has evolved quite significantly since.
